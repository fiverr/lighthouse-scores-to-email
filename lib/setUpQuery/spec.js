const setUpQuery = require('.');

describe('setUpQuery', () => {
    it('should assemble URL with parameters in query string', () => {
        expect(
            setUpQuery(
                'https://www.website.com/page?key=balue',
                {
                    categories: ['performance', 'seo'],
                    strategy: 'mobile',
                    key: 'Aa123'
                }
            )
        ).toMatchSnapshot();
    });
});
