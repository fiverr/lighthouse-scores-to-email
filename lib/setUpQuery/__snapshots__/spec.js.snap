// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setUpQuery should assemble URL with parameters in query string 1`] = `"https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=https%3A%2F%2Fwww.website.com%2Fpage%3Fkey%3Dbalue&strategy=mobile&key=Aa123&category=performance&category=seo"`;
